src/portable/project/mod.rs

```diff
@@ -112,20 +112,26 @@ const DEFAULT_SCHEMA: &str = "\
     }\n\
 ";
 
-const FUTURES_SCHEMA: &str = "\
+const FUTURE_NONRECURSIVE_ACCESS_POLICIES: &str = "\
     # Disable the application of access policies within access policies\n\
     # themselves. This behavior will become the default in EdgeDB 3.0.\n\
     # See: https://www.edgedb.com/docs/reference/ddl/access_policies#nonrecursive\n\
     using future nonrecursive_access_policies;\n\
 ";
 
-const SIMPLE_SCOPING_SCHEMA: &str = "\
+const FUTURE_SIMPLE_SCOPING: &str = "\
     # Use a simpler algorithm for resolving the scope of object names.\n\
     # This behavior will become the default in Gel 7.0.\n\
```

Comment:
We should maybe update this to say "Gel 8.0", since it isn't really happening in 7.0... I think